src/bootstrap/src/core/build_steps/tool.rs

```diff
@@ -214,6 +214,21 @@ pub fn prepare_tool_cargo(
     cargo
 }
 
+/// Copies a built tool binary with the given `name` from the build directory to the
+/// tools directory.
```

Comment:
It’s worth mentioning that this isn’t an actual copy; it’s a symbolic/soft link.

Reply:
Well, it is either a hardlink, a symlink, or an actual copy, depending on the OS. Bootstrap originally called this a "copy", but since recently it is called "copy_link". So I'll also change the name here to `copy_link_tool_bin`.